pkg/skaffold/kubernetes/owner_test.go

```diff
@@ -59,7 +59,7 @@ func TestTopLevelOwnerKey(t *testing.T) {
 			Namespace: "ns",
 			OwnerReferences: []metav1.OwnerReference{
 				{
-					Name: "dep",
+					Name: "graph",
```

Comment:
The dep -> graph changes in this file seem a bit odd, but I don't think it really matters.

Reply:
it's golang getting aggressive with it's renaming. Fixed.